coordinator/gscoordinator/coordinator.py

```diff
@@ -214,6 +214,14 @@ def ConnectSession(self, request, context):
         self._streaming_logs = True
         sys.stdout.drop(False)
 
+        # check version compatibility from client
+        if self._request.version != __version__:
+            logger.warning(
+                "Version between client and server is inconsistent: %s vs %s",
+                self._request.version,
+                __version__,
+            )
```

Comment:
We should follow the semver, and when only the patch version differ it SHOULD still could work.